packages/experimental/Expander/src/Expander.types.ts

```diff
@@ -0,0 +1,174 @@
+import { ColorValue } from 'react-native';
+import type {SyntheticEvent} from 'react-native/Libraries/Types/CoreEventTypes';
```

Comment:
Typescript will see this as "any" instead of the real type you want, since the internals of RN are in Flow.